Core/Src/main.c

```diff
@@ -31,18 +31,31 @@
 
 /* Private define ------------------------------------------------------------*/
 /* USER CODE BEGIN PD */
+
+void Adjustor_Change( const uint16_t BUTTON_PIN, char *b_on, const char increase );
+void Set_LED_Pin( char x );
+
 /* USER CODE END PD */
 
 /* Private macro -------------------------------------------------------------*/
 /* USER CODE BEGIN PM */
 
+#define ADDLED(a) ((a < 4) ? 1 : 0)
+#define MINUSLED(a) ((a > 0) ? -1 : 0)
```

Comment:
Underscore between MINUS_LED

Reply:
Fixed in [666a494](https://github.com/MuteMini/project_irrigation/pull/1/commits/666a49485adf4f24c40a469ce3da3c8f46fe1135)